packages/backend-data/API.md

```diff
@@ -12,6 +12,12 @@ import { DerivedModelSchema } from '@aws-amplify/data-schema-types';
 import { LogLevel } from '@aws-amplify/plugin-types';
 import { LogRetention } from '@aws-amplify/plugin-types';
 
+// @public
+export type AmplifyGen1DynamoDbTableMap = {
+    branchName: string;
+    modelTableNameMap?: ModelTableNameMap;

```

Comment:
```suggestion
    modelNameToTableNameMapping?: Record<string, string>;

```

or `tableNameByModelName` or `tableNameByModel`

i.e. some naming that indicates what is key vs what is value.